### ISSUE TITLE
filewatcher uses sync calls

### ISSUE THREAD

**Issue by juliangruber** [karissa/yolowatch#3]
see https://github.com/fgnass/filewatcher/blob/master/index.js#L38

i'm opening an issue there as well

**Comment by juliangruber** [comment]
i created a wip pull request for removing sync calls here: https://github.com/fgnass/filewatcher/pull/2

**Comment by okdistribute** [comment]
thank you! also just found `chokidar` which looks really nice

**Comment by juliangruber** [comment]
yeah in my experience so far that's the go to module, but maybe there's an even nicer one now
